git_machete/client.py

```diff
@@ -1049,7 +1047,7 @@ def print_line_prefix(branch_: LocalBranchShortName, suffix: str) -> None:
         sys.stdout.write(out.getvalue())
         out.close()
 
-        yellow_edge_branches = [k for k, v in edge_color.items() if v == AnsiEscapeCodes.YELLOW]
+        yellow_edge_branches = [k for k, v in sync_to_parent_status.items() if v == SyncToParentStatus.InSyncButForkPointOff]
```

Comment:
For consistency... I'd also rename `yellow_edge_branches` and `warn_on_yellow_edges` accordingly (so that edge colors are mentioned)

Reply:
I guess you meant to write `(so that edge colors are not mentioned)`, right?

Reply:
are NOT mentioned, yes! sorry